modules/apps/29-fee/keeper/msg_server.go

```diff
@@ -71,6 +73,21 @@ func (k Keeper) PayPacketFeeAsync(goCtx context.Context, msg *types.MsgPayPacket
 		return nil, types.ErrFeeModuleLocked
 	}
 
+	nextSeqSend, found := k.GetNextSequenceSend(ctx, msg.PacketId.PortId, msg.PacketId.ChannelId)
+	if !found {
+		return nil, sdkerrors.Wrapf(channeltypes.ErrSequenceSendNotFound, "channel does not exist, portID: %s, channelID: %s", msg.PacketId.PortId, msg.PacketId.ChannelId)
+	}
+
+	// only allow incentivizing of packets which have been sent
```

Comment:
Should we also check if the channel is in an `OPEN` state?

Reply:
This is a great question. The answer isn't immediately obvious. I would initially think it is a question of supporting optimistic sends, but we also need to consider the case where a channel closes. 

If a channel closes after a packet is sent, it is possible a user may want to incentivize a timeout relayer to get their packet timed out. I think that is a strong enough reason to allow incentivization on any channel state (so long as the channel exists)